x-pack/plugins/spaces/public/share_saved_objects_to_space/components/share_mode_control.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import './share_mode_control.scss';
+import React from 'react';
+import {
+  EuiCheckableCard,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormFieldset,
+  EuiIconTip,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { CoreStart } from 'src/core/public';
+import { SelectableSpacesControl } from './selectable_spaces_control';
+import { SpaceTarget } from '../types';
+
+interface Props {
+  coreStart: CoreStart;
```

Comment:
`coreStart` appears unused

Reply:
It's actually used, it is passed to `SelectableSpacesControl` along with all of the other props below
```ts
          <SelectableSpacesControl {...props} />
```

Reply:
Yep sorry I meant to update the phrasing of these comments after I went through and discovered this usage 😬